scripts/setup.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+# Provides commands for installing and uninstalling input-remapper in the system.
+# Supports using the system's `/usr/bin/python3` or the local `python3`.
+# Provides commands for cleaning up everything.
+# Supports installation of the modules in a virtual env.
+
+here="$(dirname "$0")"
```

Comment:
No benefit, I should have used `script/`, since you always run the tool from the project dir. I removed the `here` var and hard coded the `script/` path for the inject script.

Reply:
Ah, I see. But that did make sense, didn't it?

Reply:
I cleaned everything up a bit and added a better `usage` command.

This is now more of a generic development helper tool. Also, I was trying to use only the "good parts" of bash and not overdo with one-liners and funny shell magic. I hope the code is readable enough to stay maintainable here.